JASP-Engine/JASP/R/reliabilityanalysis.R

```diff
@@ -69,19 +60,24 @@ ReliabilityAnalysis <- function(dataset = NULL, options, perform = "run",
 	results[[".meta"]] <- meta
 
 	errorList <- NULL
-
-	if (is.null(resultsAlpha) && !is.null(variables) && length(variables) > 0) {
-
-		# check for errors
-		anyErrors <- .hasErrors(dataset = dataset, perform = perform,
-								type = c("infinity", "variance", "observations"),
-								observations.amount = " < 3",
-								exitAnalysisIfErrors = TRUE)
-
-		resultsAlpha <- .reliabilityResults(dataset, options, variables, perform)
-
-	} # otherwise results retrieved from state
-
+	
+	if (!is.null(variables) && length(variables) > 0) {
+		if (is.null(resultsAlpha)) { # was the main analysis retrieved from state?
```

Comment:
Is it possible for `state[["resultsAlpha"]]` to be null and `state[["confAlpha"]]` to be non-null?
In that specific case `.reliabilityResults()` would not be called, because you add both of these state items to `resultsAlpha` earlier.